packages/web-react/src/components/Toast/README.md

```diff
@@ -185,21 +188,44 @@ Alternatively, a custom icon can be used:
 | `success`     | `check-plain` |
 | `warning`     | `warning`     |
 
-### Action Link
+### ToastBar Components
 
-An action link can be added to the ToastBar component:
+The content of `Toast Bar` can be assembled from the following components:
+
+#### ToastBarMessage
```

Comment:
I think we also add an API section to subcomponents that don't have API, because even then, they have escape hatches.